pkg/kube/resources_test.go

```diff
@@ -46,9 +50,12 @@ func TestGetMultipleResourceFromSingleFile(t *testing.T) {
 func TestGetResourceFromAPI(t *testing.T) {
 	k8s := test.SetupTestAPI()
 	k8s = test.SetupAddDeploys(k8s, "test")
-	resources, err := CreateResourceProviderFromAPI(k8s)
+	resources, err := CreateResourceProviderFromAPI(k8s, "test")
```

Comment:
Is there value in also testing that CreationTime attribute exists, or is that not super important in this context?

Reply:
Good call!